Sources/HaCWebsiteLib/ViewModels/ImageHero.swift

```diff
@@ -1,15 +1,35 @@
 import HaCTML
+import Foundation
 
 struct ImageHero {
   let background: BackgroundType
   let imagePath: String
   let alternateText: String
   let destinationURL: String?
+  let dateToDisplay: Date?
+  let textColor: Color
+
+  init(
```

Comment:
Can still take advantage of default initialisers here, can put the default values up here